web-server/v0.4/src/services/explore.js

```diff
@@ -19,3 +19,26 @@ export default async function querySharedSessions(params) {
     },
   });
 }
+export async function deleteSharedSessions(params) {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
We can remove this commented line since all of the declared vars are being used in the next line.